plugins/opener/src/reveal_item_in_dir.rs

```diff
@@ -35,10 +35,47 @@ pub fn reveal_item_in_dir<P: AsRef<Path>>(path: P) -> crate::Result<()> {
     Err(crate::Error::UnsupportedPlatform)
 }
 
+/// Reveal the paths the system's default explorer.
+///
+/// ## Platform-specific:
+///
+/// - **Android / iOS:** Unsupported.
+pub fn reveal_items_in_dir<P: AsRef<Path>>(paths: &[P]) -> crate::Result<()> {
+    let mut path_bufs = vec![];
+
+    for path in paths.iter() {
+        let path = path.as_ref().canonicalize()?;
+        path_bufs.push(path);
+    }
```

Comment:
This way we could take all combination of `&[&Path]` `[PathBuf]`s

```suggestion
pub fn reveal_items_in_dir<I, P>(paths: I) -> crate::Result<()>
where
    I: IntoIterator<Item = P>,
    P: AsRef<Path>,
{
    let mut canonicalized = vec![];

    for path in paths {
        let path = path.as_ref().canonicalize()?;
        canonicalized.push(path);
    }
```

Reply:
Applied 🫡